gui/src/main/java/org/verapdf/gui/tools/ProcessingType.java

```diff
@@ -25,6 +25,35 @@ public boolean isFeatures() {
 		return this == FEATURES || this == VALIDATING_AND_FEATURES;
 	}
 
+	public static ProcessingType fromString(String string) throws IllegalArgumentException{
+		if(string == null)
+			throw new IllegalArgumentException("String to be parsed into ProcessingType cannot be null");
+
+		switch(string) {
```

Comment:
as all String constants uses in ProcessingType and ProcessingType will be placed out of gui package, then all of this constants should be moved to ProcessingType class.